src/main/java/com/commercetools/project/sync/cartdiscount/CartDiscountSyncer.java

```diff
@@ -51,8 +53,26 @@ public static CartDiscountSyncer of(
 
     final CartDiscountSyncOptions syncOptions =
         CartDiscountSyncOptionsBuilder.of(targetClient)
-            .errorCallback(LOGGER::error)
-            .warningCallback(LOGGER::warn)
+            .errorCallback(
+                (exception, newResourceDraft, oldResource, updateActions) -> {
```

Comment:
What is your plan for covering those, maybe you could create an utility method and test it also format part looks similar for all Syncers so it would be nice to avoid duplicates?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/158/commits/739cf2461761fa4f3aab6899b8eb5996143ea19e

Reply:
Still not covered though. I have to think of some other tests to raise the coverage....

Reply:
How was it covered before?

Reply:
It was not covered 😄 I think Codecov just didn't care before. Now it cares as I rewrote it as a lambda function.